addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -449,47 +486,61 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
             }
             if ($snippet) {
                 $snippet[0].normalize();
-                this.trigger_up('activate_snippet', {
-                    $snippet: $snippet,
-                    previewMode: false,
-                    onSuccess: () => textOptionsPostActivate($snippet),
+                this._activateSnippet($snippet, false).then(() => {
+                    textOptionsPostActivate($snippet);
                 });
                 this.options.wysiwyg.odooEditor.historyStep();
+                return true;
             } else {
                 this.notification.add(
                     _t("Cannot apply this option on current text selection. Try clearing the format and try again."),
                     { type: 'danger', sticky: true }
                 );
             }
+            return false;
         }
-    },
+    }
     /**
+     * Used to adjust the highlight effect when the text content is edited.
+     *
+     * TODO: Should be directly replaced by `switchTextHighlight()` in master
+     * (left in stable for compatibility).
```

Comment:
Looks like a rebase fail on top of the recently merged commit of Mehdi. Have a look :pray: